app/js/arethusa.core/directives/fullHeight.js

```diff
@@ -17,9 +17,6 @@ angular.module('arethusa.core').directive('fullHeight', [
           var fullHeight = body.height();
           element.height(fullHeight - margin - padding - additionalBorder);
         }
-        win.bind('resize', function() {
-          resize();
-        });
```

Comment:
so, without this, the tree stays the same size when the window resizes?  We might want to add an API call to allow embedding code to resize the tree then.

Reply:
Interestingly, the tree still resizes without this code. With this code, resizing the screen horizontally causes the tree to grow vertically. That meant that every time the user resized the screen the tree would grow longer and longer.

Reply:
ok, thanks. I think it's fine, and if it turns out we need to add something to the api, we can.